### PR TITLE
Issue #933: Describe a good PR

### PR DESCRIPTION
**Pull request by timstallmann**
This pull request:
* Adds a description of a good pull request to the developer docs repo.

I'm not sure if this is too concise or if we need more information.

### CODE REVIEW

File: git.md [hunk 1]
```diff
@@ -21,6 +21,15 @@ We also have a few in-house additions to this:
 
 ## Creating a good pull request
```
Comment (kostajh) [inline git.md]: @timstallmann can you add a note about `[WIP]` pull request and convention for those?

File: git.md [hunk 2]
```diff
@@ -21,6 +21,15 @@ We also have a few in-house additions to this:
...
 
+A pull request is an important piece of communication, both to other developers reviewing your code and to future developers who need to understand the history of the project.
+ A good pull request description should:
+ 
+* Give a concise summary of what changes are introduced to the codebase and/or functionality in the pull request. Often we do this using a bullet list.
+* List any steps which will necessary to deploy the pull request, both locally and in production -- this should have all the information necessary to deploy.
+* Give any specific instructions that are necessary for testing. Include links to URLs for testing whenever you can!
+* Specify any specific issues that you want developers doing peer review to be sure to investigate thoroughly.
+* Give a timeframe for review.
+
```
Comment (kostajh) [inline git.md]: A few more things to add:

- [ ] Assign a reviewer in GitHub UI
- [ ] If applicable, update the corresponding Redmine issue and set the GitHub PR field there with a link to the PR, and assign a reviewer in Redmine

File: git.md [hunk 3]
```diff
@@ -21,6 +21,22 @@ We also have a few in-house additions to this:
 
 ## Creating a good pull request
 
+A pull request is an important piece of communication, both to other developers reviewing your code and to future developers who need to understand the history of the project.
+Like commit messages, pull request titles should start with `Issue #___: ` and be a one-line summary of what the PR does. **If your pull request is not ready for review**, mark it
+as a work-in-progress by putting `[WIP]` in front of the title.
```
Comment (kostajh) [inline git.md]: Could we also add a line like, "Once your PR is ready for review, be sure to remove `[WIP]` and let the reviewer know." This has been an issue in the past

File: git.md (hunk 1 above)
Comment (timstallmann) [inline git.md]: Done!

File: git.md (hunk 3 above)
Comment (timstallmann) [inline git.md]: @kostajh done!

**Comment by kostajh** [comment]
Hey, I just remembered something else. When the reviewer is done, and merges the PR, my preference is for them to click the "Delete branch" button to help with decreasing clutter in the repo. Does that sound OK to you?

**Comment by kostajh** [comment]
Just discussed the above with @timstallmann -- we both agree to drop that suggestion.

**Comment by kostajh** [comment]
Merged manually. Thanks @timstallmann!

**Pull request closed without merging**